linera-rpc/src/config.rs

```diff
@@ -103,8 +102,8 @@ pub type ValidatorPublicNetworkConfig = ValidatorPublicNetworkPreConfig<NetworkP
 /// The network configuration for all shards.
 #[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
 pub struct ValidatorInternalNetworkPreConfig<P> {
-    /// The name of the validator.
-    pub name: ValidatorName,
+    /// The public key of the validator.
+    pub name: ValidatorPublicKey,
```

Comment:
public_key: ValidatorPublicKey

Reply:
Done in [Fix missed renames](https://github.com/linera-io/linera-protocol/pull/3372/commits/1c4535584e4c5794a6c793a09490c90608de4c9a).